source/commands/stats.js

```diff
@@ -10,7 +10,7 @@ const options = [
 	{
 		type: "User",
 		name: "bounty-hunter",
-		description: "Whose stats to check, BountyBot for the guild's stats",
+		description: "Whose stats to check (defaults to yourself); input BountyBot for the server stats",
```

Comment:
This parenthetical feels...off. I think this would fit better after the semicolon right now. Maybe convert the semicolon to a period, and separate the default/BB bit by a semicolon?

Reply:
This `Whose stats to check. Input BountyBot for the server stats; defaults to yourself`?

I personally don't like periods at the end of sentence fragments. What about the following?

`Whose stats to check; input BountyBot for the server stats; none defaults to yourself`

Reply:
Here's how it actually shows in the UI
![image](https://github.com/Imaginary-Horizons-Productions/BountyBot/assets/6503151/cc21c4cf-67ff-4146-b9fe-dcdb28427ebd)
Maybe it should only be the special cases

Reply:
Yeah, I personally prefer periods on sentence fragments for semi-declarative stuff like this, but could see how it could be grating. I like that first one that you don't, but consistency is more important than anything else. I do like that second option from before the screenshots.

Reply:
Done in https://github.com/Imaginary-Horizons-Productions/BountyBot/pull/23/commits/62e6aa7fe3b75ffafe650a6bd8e8e235c9dd859f